README.md

```diff
@@ -1 +1,22 @@
-# llm-translator
\ No newline at end of file
+# Translatorbot
+
+Live speech-to-speech translation in a Daily call.
+
+This demo has two parts. `translatorbot` contains the daily-python code that powers the translation. `frontend` contains a lightly modified version of Daily's [React demo app](https://github.com/daily-demos/custom-video-daily-react-hooks) that allows participants to select which audio and subtitles they receive.
+
+## `translatorbot`
```

Comment:
When wrapping this up, let's add a screenshot of what the app looks like and some more descriptive README text here. E.g., should they sign up for a Daily account to run this locally? Anything else that is nice to know?